src/layers/doubleSymLayer.jl

```diff
@@ -39,21 +39,18 @@ end
 
 function apply(this::DoubleSymLayer{T},theta::Array{T},Yin::Array{T,2},doDerivative=true) where {T<:Number}
 
-    #QZ = []
-    tmp = Array{Any}(2)
     nex = div(length(Yin),nFeatIn(this))::Int
     Y   = reshape(Yin,:,nex)
 
     theta1,theta2,theta3,theta4 = splitWeights(this,theta)
     Kop    = getOp(this.K,theta1)
     KY     = Kop*Y
-    KY,dummy,tmp[1] = apply(this.nLayer,theta4,KY)
-    Yt     = KY
+	tmp    = (doDerivative) ? copy(KY) : (T)[]
```

Comment:
Hi Lars, 
You now only store KY in tmp whereas before we stored KY as tmp[2] and had some other Ys stored in tmp[1] corresponding to the other Ys in batchNormNN.. is this all we need now? If so I can fix the conflicts and merge yours in.

Reply:
Hi David, there is no need anymore to store Ys. Note that doing this doubles the memory. Also, from KY I can re-compute everything thats needed without convolutions.  It would be great if you could help with the conflicts! Thanks

Reply:
Awesome. That will help a lot with the memory . Thanks!

Reply:
Actually, while we're at it: More memory can be saved by writing a specialized bach norm layer (see MATLAB here: https://github.com/XtractOpen/Meganet.m/commit/c40b9a74cab1c59a5353958ad108e6d0e39c6e3a) I know you've recently combined them in batchNorm.jl but there you keep some tmp variables, right? Not needed for batch norm, I think!